cmd/notation/cache.go

```diff
@@ -0,0 +1,235 @@
+package main
```

Comment:
I may suggest change the file name to something like `signature-cache` to be specific.

Reply:
notation is about signature :P